content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -34,18 +34,10 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
 
 For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
 
-### Multiple LNS connections for Sigfox and Actility
-
-The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
-
-![All devices](/images/release-notes/multiple_lns_connection.png)
-
-For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
-
-
+### Device Management
 ### Revised LWM2M device registration
```

Comment:
This and the following headings should have heading level 4 ####

Reply:
Done